samples/azure-service-bus-netstandard/native-integration-pub-sub/ASBS_1/NativeShared/TopologyManager.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace NativeSender
+{
+    using System;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.ServiceBus;
+    using Microsoft.Azure.ServiceBus.Management;
+
+    public static class TopologyManager
+    {
+        public static async Task CreateSubscription(string connectionString, string name, SqlFilter sqlFilter, string topicPath = "bundle-1", string ruleName = "rule")
```

Comment:
The default value for a rule name in the following way

```c#
ruleName = "rule"
```

is going to break the moment a subscriber has more than one event to subscribe to. While the sample is not a production-grade code, we should strive to lead in the right direction. Making a rule name mandatory is a good way to make readers think about it rather than assume it can be ignored.